org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/discovery/ZigBeeDiscoveryService.java

```diff
@@ -128,9 +133,11 @@ public void startScan() {
             }
 
             logger.debug("Starting ZigBee scan for {}", coordinator.getUID());
-            coordinator.scanStart();
+            coordinator.scanStart(SEARCH_TIME);
         }
 
+        Runnable searchFinishedIndicator = () -> searchStarted = false;
```

Comment:
Does `searchStarted` need to be synchronized? See also my above.

Reply:
That's why I made the the variable `volatile` :-) See https://www.javamex.com/tutorials/synchronization_volatile.shtml